# Bepaling van soortelijke warmte van water


# Introductie

Water heeft een enorme buffercapaciteit: je moet veel energie toevoegen om de temperatuur van water een graad te verwarmen. In dit practicum gaan we de soortelijke warmte van water bepalen door een bekende hoeveelheid water te verwarmen met een bekende hoeveelheid energie, en de temperatuurstijging te meten.

# Theorie

De soortelijke warmte $c$ van een stof is de hoeveelheid energie die nodig is om de temperatuur van één kilogram van die stof met één graad Celsius (of één kelvin) te verhogen. De fundamentele relatie wordt gegeven door:

$$
Q= {mc}{ \Delta T}
$$

Daardoor wordt c gegeven door: 

$$
c=\frac{Q}{m \Delta T}
$$

Waarbij: 


Q de totale toegevoerde warmte-energie is in Joules (J), m de massa van de stof is in kilogram (kg), c is de specifieke warmte is in Joules per kilogram per kelvin, (delta)T = T_f - T_i de temperatuurverandering is in graden Celsius of kelvin.

# Methode en materialen

## Ontwerp
Een waterbad met bekende massa aan water wordt verwarmd met een elektrisch verwarmingselement dat een bekende hoeveelheid energie levert.
De temperatuur van het water wordt gemeten met een temperatuursensor.
Door de temperatuurstijging als functie van de tijd te meten kan de soortelijke warmte van water worden berekend.


## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Maatbeker
- Weegschaal
- Water
- Elektrisch verwarmingselement ($10 \mathrm{\Omega}$, $10 \mathrm{W}$)
- Voedingsbron 
- Thermometer of temperatuursensor
- Stopwatch of timer

```{figure} figures/cwater_setup.jpg
:width: 70%

Een schematische weergave van de opstelling
```

## Procedure
```{exercise}
1. Beschrijf de procedure die nodig is om de soortelijke warmte van water te bepalen.
2. Geef jouw beschrijving van de procedure aan een ander team, zij verzamelen voor jou de data op basis van de gegevens.
3. Voer het experiment van een ander team uit op basis van hun beschrijving.
4. Analyseer de data die het andere groepje voor jou heeft verzameld.
```


## Veiligheid
We maken gebruik van een $10 \mathrm{\Omega}$, $10 \mathrm{W}$ weerstand.
Deze wordt snel heet.
De bronspanning mag dan ook alleen aan wanneer de weerstand in het water zit.
Raak de weerstand niet aan tijdens het experiment.
Omdat de weerstand in het water zit, kunnen we wel het elektrisch vermogen hoger zetten zonder dat de weerstand oververhit raakt.
Het maximaal vermogen mag $40 \mathrm{W}$ zijn.
Daarbij moet de roerder wel aanstaan om de warmte goed te verdelen.

## Data analyse
#Procedure

1. Meet de massa van het water op een weegschaal (omdat de onze kapot was, gebruikten we de volumebenadering waar 1 ml = 1 g)
2. Meet de begintemperatuur van het water in graden Celsius
2. Plaats de weerstandsverwarmer in het water en verander de spanning en stroomsterkte en start een timer voor 100 seconden
3. Plaats een thermometer in het water en meet de eindtemperatuur na 100s
4. Herhaal dit voor 4 verschillende spanningen en stroomsterktes (dit verandert het vermogen)
5. Verwerk de data in NumPy-arrays, we zullen later de formule uit de theorie gebruiken om de soortelijke warmtecapaciteit van water te berekenen.

In [1]:
#1. Bereken het elektrisch vermogen voor elke meting
#P = V * I  # Vermogen in Watt (W)

# 2. Bereken de toegevoerde energie voor elke meting
#Q = P * t  # Energie in Joule (J), waarbij t = 100 s

# 3. Bereken de temperatuurverandering voor elke meting
#ΔT = T_f - T_i  # in °C of K

# 4. Bereken de soortelijke warmte van water voor elke meting
#c_water = Q / (m * ΔT)  # in J/(kgK)

# Resultaten


In [4]:
# Hier de data en de analyse
import numpy as np
import matplotlib.pyplot as plt

mass_water = 0.250 #kg
expected_c_water = 4186 #theoretical

Voltage = np.array([12.6, 15.0, 16.7, 19.3]) #V
Current = np.array([1.30, 1.55, 1.72, 1.80]) #A
Time = np.array([100, 100, 100, 100,]) #s
Temp_i = 19.4 #deg C - kept temp in degree C because delta T is unitless
Temp_f = np.array([21.0, 21.3, 21.9, 22.8]) #deg C


Power= Voltage * Current #W
Energy = Power * Time #J
Delta_T = Temp_f - Temp_i #deg C
c_water_calculated = Energy / (mass_water * Delta_T) #J/kgK

c_mean=np.mean(c_water_calculated)
print("De gemiddelde soortelijke warmte is:", c_mean)
plt.savefig("soortelijke_warmte_plot.png")

De gemiddelde soortelijke warmte is: 4418.158916408665


<Figure size 640x480 with 0 Axes>

In [3]:
# Sla figuren op met  
# 
# plt.savefig("figures/naam.png", dpi=450)



# Discussie en conclusie

Onze gemiddelde waarde uitendelijk was 4418 J/kgK en de theoretische waarde was, 4186 dus uiteindelijk waren hadden wij een afwijking van 7%. Dit lijk best redelijk omdat er weining metingen gemaakt waren door tijdsnood. Ook waren er approximations gemaakt, bijvoorbeeld dat de tijd precies 100s was en geen ms meer of onder, dus dit is natuurlijk een random error die meespeelt bij het afwijking. Toch is de gemmidelde waarde goed genoeg. 